lib/msal-browser/README.md

```diff
@@ -22,6 +22,9 @@
 
 ## About
 
+>[!NOTE]
+>MSAL.js 2.0 with Authorization Code Flow **is not yet available for B2C tennant** (comming soon) 
```

Comment:
This syntax won't work on Github, unfortunately.  I would maybe do something like this:

```md
**Important:** MSAL.js 2.0 with Authorization Code Flow is not yet available for B2C tenants (coming soon).
```